components/DaysOfTheWeek/DaysOfTheWeekContainer.jsx

```diff
@@ -0,0 +1,122 @@
+import { useState } from "react";
+import { Text, Box, SimpleGrid, GridItem, Flex } from "@chakra-ui/react";
+import DaysOfWeekButton from "./DayOfTheWeekButton";
+import LinkButton from "../LinkButton/LinkButton";
+
+export default function DaysOfTheWeekContainer ({ setNumberOfDays, onSaveEvent }) {
+
+  // initial state for days of the week has info if it's selected or not (instead of having 2 separate states)
+  const [ daysOfTheWeek, setDaysOfTheWeek ] = useState([
+    {
+      id: 0,
+      day: "Monday",
+      isSelected: false
+    },
+    {
+      id: 1,
+      day: "Tuesday",
+      isSelected: false
+    },
+    {
+      id: 2,
+      day: "Wednesday",
+      isSelected: false
+    },
+    {
+      id: 3,
+      day: "Thursday",
+      isSelected: false
+    },
+    {
+      id: 4,
+      day: "Friday",
+      isSelected: false
+    },
+    {
+      id: 5,
+      day: "Saturday",
+      isSelected: false
+    },
+    {
+      id: 6,
+      day: "Sunday",
+      isSelected: false
+    }
+  ]);
+
+  // state to activate and disable save button (link button)
+  const [ saveButtonActive, setSaveButtonActive ] = useState(true);
+
+  // on click, the buttom will change colour and the state will be updated to include selected buttons
+  const handleClick = (e) => {
+    e.preventDefault();
+
+    let updatedData = [...daysOfTheWeek];
+
+    // based on the day name from innerHTML of each button, update the state if selected or unselected
+    updatedData.map(item => {
+      if (item.day === e.target.innerText) {
+        item.isSelected = item.isSelected ? false : true;
+      }
+    })
+    setDaysOfTheWeek(updatedData);
+
+    // activate Save button (link button) when something is selected
+    const areDaysSelected = updatedData.find(item => item.isSelected) ? false : true;
+    setSaveButtonActive(areDaysSelected);
+
+    // get answer to how many days a week user works
+    const workingDaysNumber = updatedData.filter(item => item.isSelected).length;
+    console.log(workingDaysNumber)
```

Comment:
Happy to leave this `console.log` message here for now, as it's useful for debugging.

But we will want to set a reminder for ourselves to remove this prior to release :+1:

Reply:
Let's set up a reminder to remove all console.log from the project at once.